README.md

```diff
@@ -226,6 +226,14 @@ await client.logon();
 await client.logoff();
 ```
 
+## Creating new instances
+Creates and returns a new instance of the specified entity
+
+```js
+const template = await client.newInstance('xtk:persist|nms:delivery', { xtkschema: 'nms:delivery', {label: 'Test', messageType: '0', isModel: '1' } });
```

Comment:
Indentation issues (missing spaces). Also, messageType and isModel are supposed to be numeric values. Yes it works with strings, but this is an example showing up the best practices, so we should pass numeric values here